scripts/manpage_generation.sh

```diff
@@ -6,7 +6,11 @@ MANPAGE_DIR="man"
 VER=$(grep -oP '^Version:\s+\K\S+' packaging/convert2rhel.spec)
 
 # Generate a file with convert2rhel synopsis for argparse-manpage
-python -c 'from convert2rhel import toolopts; print("[synopsis]\n."+toolopts.CLI.usage())' > man/synopsis
+/usr/bin/python3.10 -c 'import sys;print(sys.path)'
+/usr/bin/python3.10 -c 'from convert2rhel import toolopts; print("[synopsis]\n."+toolopts.CLI.usage())' > man/synopsis
+
+/usr/bin/python3.10 -m pip install argparse-manpage six pexpect
+/usr/bin/python3.10 -m pip show --files argparse-manpage
```

Comment:
Also, don't forget to remove those debug lines later! 

```suggestion
/usr/bin/python3.10 -c 'from convert2rhel import toolopts; print("[synopsis]\n."+toolopts.CLI.usage())' > man/synopsis

/usr/bin/python3.10 -m pip install argparse-manpage six pexpect
```